syntax = "proto3";
package mizar;

import "google/protobuf/empty.proto";

// service
service BuiltinsService {
    rpc CreateService(BuiltinsMessage) returns (google.protobuf.Empty) {}
    rpc UpdateService(BuiltinsMessage) returns (google.protobuf.Empty) {}
    rpc CreatePod(BuiltinsMessage) returns (google.protobuf.Empty) {}
    rpc CreateNode(BuiltinsMessage) returns (google.protobuf.Empty) {}
}

message BuiltinsMessage {
  string Name = 1;
  string Body = 2;
  string Spec = 3;
}